### PR TITLE
Enhancement sumo interface

### PR DESCRIPTION
**Pull request by DavidMansolino**
Linked with omichel/webots#3877

https://www.cyberbotics.com/doc/automobile/sumo-interface?version=enhancement-sumo-supervisor

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:+1:

File: automobile/sumo-interface.md [hunk 1]
```diff
@@ -17,7 +17,9 @@ configuration files (.netccfg and .sumocfg).
 Then the interface will automatically start SUMO and run it in synchronization
 with Webots time. Each time a new vehicle enter the SUMO simulation, it will be
 created in Webots too and its position and orientation will be continually
-updated. If the simulation contains traffic lights, the name of the
+updated. The vehicle DEF name is set to `SUMO_VEHICLEX`, `X` been the vehicle number (starting from 0). If some vehicles whose DEF name is `SUMO_VEHICLEX` are already present in the world at the simulation start, then the interface will automatically use them before creating new vehicles, this can be usefull to avoid real-time addition of vehicles (wich can make the simulation speed drop for a very short time). If some vehicles whose DEF name is `WEBOTS_VEHICLEX` (X been the vehicle number starting from 0) are present in the simulation, the interface will automatically add them and update their position and orientation in SUMO in order to close the loop.
```
Comment (stefaniapedrazzi) [inline automobile/sumo-interface.md]: I think you have to use ``being`` and not ``been`` in the sentence ``'X' been the vehicle number```.
And personally I would use ``with 'X' being ..``.

typo: usefull ->useful

Comment (DavidMansolino) [inline automobile/sumo-interface.md]: > And personally I would use with 'X' being ...

Yes, that's sound indeed better, thank you.

**Comment by DavidMansolino** [comment]
:+1:

**Comment by stefaniapedrazzi** [comment]
:+1:

File: automobile/sumo-interface.md [hunk 2]
```diff
@@ -17,7 +17,9 @@ configuration files (.netccfg and .sumocfg).
 Then the interface will automatically start SUMO and run it in synchronization
 with Webots time. Each time a new vehicle enter the SUMO simulation, it will be
 created in Webots too and its position and orientation will be continually
-updated. If the simulation contains traffic lights, the name of the
+updated. The vehicle DEF name is set to `SUMO_VEHICLEX`, with `X` being the vehicle number (starting from 0). If some vehicles whose DEF name is `SUMO_VEHICLEX` are already present in the world at the simulation start, then the interface will automatically use them before creating new vehicles, this can be useful to avoid real-time addition of vehicles (wich can make the simulation speed drop for a very short time). If some vehicles whose DEF name is `WEBOTS_VEHICLEX` (with `X` being the vehicle number starting from 0) are present in the simulation, the interface will automatically add them and update their position and orientation in SUMO in order to close the loop.
```
Comment (omichel) [inline automobile/sumo-interface.md]: Typo: wich => which

File: automobile/sumo-interface.md [hunk 3]
```diff
@@ -61,10 +63,12 @@ are: the `Supervisor` itself, the traci class in order to get information about
 the traci context and the net class to get information about the network. The
 second one is called at each SUMO step and the argument is the time step.
 
-Such plugin can be used for example to change traffic light state in SUMO. Or
-for creating a vehicle not controlled by SUMO but with a regular controller in
-Webots and then close the loop with SUMO.
+Such plugin can be used for example to change traffic light state in SUMO.
```
Comment (omichel) [inline automobile/sumo-interface.md]: Either "Such **a** plugin" or "Such plugin**s**".

**Comment by DavidMansolino** [comment]
:+1:

**Comment by omichel** [comment]
:+1:

**Pull request merged**